tests/integration/nm/linux_bridge_test.py

```diff
@@ -154,7 +154,12 @@ def _modify_ports(ports_state):
 @nmclient_context
 def _get_bridge_current_state():
     nmdev = nm.device.get_device_by_name(BRIDGE0)
-    return nm.bridge.get_info(nmdev) if nmdev else {}
+    state = nm.bridge.get_info(nmdev) if nmdev else {}
+    if state:
+        slaves = state.get(LB.CONFIG_SUBTREE, {}).get(LB.PORT_SUBTREE, [])
+        slaves.sort(key=lambda d: d[LB.Port.NAME])
```

Comment:
This seems to be an unrelated fix. Can you extract it to its own commit? (with an explanation in the commit message)

Reply:
Done, thanks!